packages/lib/survey/service.ts

```diff
@@ -257,6 +288,13 @@ export async function updateSurvey(updatedSurvey: TSurvey): Promise<TSurvey> {
       if (currentSurvey.triggers.find((t) => t === trigger)) {
         continue;
       } else {
+        const actionClassId: string = getActionClassIdFromTrigger(actionClasses, trigger);
```

Comment:
I think it would be better readable to combine the two revalidation iterations and add this below the trigger iterations:

```ts
for (trigger of [...newTriggers,...removedTriggers]) {
  // Revalidate new actionClasseId
  surveyCache.revalidate({
    actionClassId,
  });
}
```

what do you think?

Reply:
If we choose this, we should also do this for actionClasses below